vignettes/mtcars_gradual.Rmd

```diff
@@ -0,0 +1,295 @@
+---
+title: "Demo with mtcars, complex"
+vignette: >
+  %\VignetteIndexEntry{Demo with mtcars}
```

Comment:
The vignette itself looks good to me! But can we align the different titles and the filename? This seems more than a demo to me -- perhaps a walkthrough?

Reply:
I had a look and found that the original `mtcars` vignette didn't really add anything anymore, and I found that it would be useful to replace it with this one.
This file is now called `mtcars.Rmd` and titled `Functionality walkthrough (using the mtcars dataset)`, if you're fine with that. I could also rename the file as `functionality` or something like that?